packages/front-end/pages/saved-groups/[sgid].tsx

```diff
@@ -281,12 +281,16 @@ export default function EditSavedGroupPage() {
               <div className="mr-2">Projects:</div>
 
               <div>
-                <div className={"d-flex align-items-center"}>
-                  <ProjectBadges
-                    projectIds={savedGroup.projects}
-                    resourceType="saved group"
-                  />
-                </div>
+                {savedGroup.projects?.length ? (
+                  <div className={"d-flex align-items-center"}>
+                    <ProjectBadges
+                      projectIds={savedGroup.projects}
+                      resourceType="saved group"
+                    />
+                  </div>
+                ) : (
+                  <strong>None</strong>
```

Comment:
Hmm maybe the logic should be more like how we do it on the ID Lists list page on second thought
```
                            {(s?.projects?.length || 0) > 0 ? (
                              <ProjectBadges
                                resourceType="saved group"
                                projectIds={s.projects}
                                className="badge-ellipsis short align-middle"
                              />
                            ) : (
                              <ProjectBadges
                                resourceType="saved group"
                                className="badge-ellipsis short align-middle"
                              />
                            )}
```
I think we want it to render as `All Projects` instead of none

Reply:
yeah sounds right

Reply:
Wait, why do we need this to be in a ternary? Shouldn't we just make `ProjectBadges` do that if `projectIds` is an empty list? (probably out of scope for right now cause I want to get your changes merged, but it's weird to me to have this behavior)

Reply:
I'm fairly sure that's from this line. Either way we can deal with that behavior separately
https://github.com/growthbook/growthbook/blob/msamper/project-save-groups/packages/back-end/src/models/SavedGroupModel.ts#L151